src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* **Breaking Change** Removed support for ParseStateValues if the state neither
```

Comment:
The wording could use better phrasing, to avoid unnecessarily confusing folks about what is the breaking change, given 99.99% users won't be affected with this.!

1. The ability to parse random objects into Attributes using Reflection was introduced only in 1.5.0. So need to highlight that this is removing a capability added in 1.5.0.  
(An alternative would be to treat this as a mistake and do 1.5.1 removing this capability).

Reply:
💯